web/src/settings_users.ts

```diff
@@ -554,12 +570,6 @@ export function redraw_active_users_list(): void {
 }
 
 function start_data_load(): void {
-    loading.make_indicator($("#admin_page_users_loading_indicator"), {
-        text: $t({defaultMessage: "Loading…"}),
-    });
-    loading.make_indicator($("#admin_page_deactivated_users_loading_indicator"), {
-        text: $t({defaultMessage: "Loading…"}),
-    });
```

Comment:
Why do you remove all of this loading indicator code? 

It seems very unlikely to be correct. Likely you instead want to call the function to hide those indicators inside `render_table`, and probably also inside an `error` handler for the `channel.post` call.

Reply:
I removed these `loading indicators` because you mentioned, `it might be nice to have the 'Users' table load immediately and just show loading indicators in the relevant columns` where availability data is not available initially, and then update those values via a call to the new API endpoint. Therefore, these loading indicators seemed unnecessary. 

Please correct me if I misunderstood your suggestion.

Please have a look at the new behaviour .
Thanks!
<details>
<summary>New Behaviour: </summary>

https://github.com/user-attachments/assets/3ddb36ce-ff34-4097-9fc1-8d15b0d4605c


</details>

Would it be a good idea to keep them as they were previously?

Reply:
OK but you didn't implement the new location for loading indicators.

Look, if the user is waiting for the app to fetch data from somewhere, there should be a loading indicator, rather than it just silently not showing the data. This is basic UX design. I suggested a nice improvement might be to have the loading indicator, rather than being global, be just in the place where there's data missing. But under no circumstances does it make sense to just remove an existing working set of loading indicators.

Reply:
The video does seem to have indicators in the new place, but they're just the word "Loading..." -- you should use the `loading.ts` spinners that look nice. I think that not being there was the source of my confusion.

Reply:
@timabbott, I have restored the previous spinners. However, I haven't yet implemented the spinner alongside `Loading...` in each row. I was trying to figure out how to add that spinner, but it was taking time. I can either add a commit or create a follow-up PR—whichever you prefer—to include the spinner in each row once my college exams are over.

Thanks!

Reply:
Yeah that can be a follow-up project.